servers/graphql-kotlin-server/src/main/kotlin/com/expediagroup/graphql/server/extensions/responseExtensions.kt

```diff
@@ -33,14 +34,28 @@ fun ExecutionResult.toGraphQLResponse(): GraphQLResponse<*> {
     return GraphQLResponse(data, filteredErrors, filteredExtensions)
 }
 
+/**
+ * Turn a thrown exception into a graphql-java [GraphQLError] that we can return in the response
+ */
+fun Throwable.toGraphQLError(): GraphQLError {
+    return if (this is GraphQLError) {
```

Comment:
This is the big change. Everything stems from what we no longer need to do around error mapping